frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -29,12 +29,12 @@ export enum OnboardingStepKey {
 export type AllOnboardingSteps = OnboardingStep[]
 export type OnboardingStep = JSX.Element
 
-export const getProductUri = (productKey: ProductKey, featureFlags: FeatureFlagsSet): string => {
+export const getProductUri = (productKey: ProductKey, featureFlags?: FeatureFlagsSet): string => {
     switch (productKey) {
         case ProductKey.PRODUCT_ANALYTICS:
-            return featureFlags[FEATURE_FLAGS.REDIRECT_WEB_PRODUCT_ANALYTICS_ONBOARDING] === 'test'
+            return featureFlags && featureFlags[FEATURE_FLAGS.REDIRECT_WEB_PRODUCT_ANALYTICS_ONBOARDING] === 'test'
                 ? combineUrl(urls.webAnalytics(), { onboarding_completed: true }).url
-                : combineUrl(urls.insights(), { onboarding_completed: true }).url
+                : combineUrl(urls.events(), { onboarding_completed: true }).url
```

Comment:
I think redirecting to /insights performed better than redirecting to /events.

Reply:
ah! bad merge